index.html

```diff
@@ -12,5 +12,75 @@
       href="assets/css/style.css"
     />
   </head>
-  <body></body>
+  <body>
+    <div class="container">
+      <section class="hero">
```

Comment:
By convention `<section>` tag requires heading tag (h1,h2...). Please use header,main,aside tags ...
I would call this header. Convetion isn't must it's only good practise.